src/views/Main/MenuDropdown/index.js

```diff
@@ -4,6 +4,14 @@ import { connectAppProvider } from '../../../components/AppProvider';
 import { getComponentsByEnabledModules } from '../../../services/modules';
 import { MenuDropdown } from './MenuDropdown';
 
-export default connectAppProvider(
-  ({ env: { enabled_modules: modules } }) => ({ modules: getComponentsByEnabledModules(modules) }),
-)(withNamespaces()(withRouter(MenuDropdown)));
+export default withRouter(
+  connectAppProvider(
+    ({
```

Comment:
Please put `connectAppProvider()` parameters in a variable to improve readability.

Maybe something like: 
```js
const findMeABetterName = ({ env: { enabled_modules: modules } }) => ({ modules: getComponentsByEnabledModules(modules) });

export default withRouter(
  connectAppProvider(findMeABetterName)(
    withNamespaces()(MenuDropdown),
  ),
);
```

Reply:
Done